convert/convert.go

```diff
@@ -49,7 +49,12 @@ func trimTokens(tokens hclwrite.Tokens) hclwrite.Tokens {
 	return tokens[startIndex : len(tokens)-endIndex]
 }
 
-func buildMapExpression(tokens hclwrite.Tokens) string {
+func buildMapExpression(tokens hclwrite.Tokens, tfVersion int) string {
+	if tfVersion >= 15 {
```

Comment:
Since `tomap` was introduced in version 0.12 maybe we should change that.
```suggestion
	if tfVersion >= 12 {
```

Reply:
I wouldn't change older version generation at this point.